_sass/blocks/_landing.scss

```diff
@@ -0,0 +1,136 @@
+.landing-section_open,
+.landing-section {
+  padding-bottom: $base-padding;
+  padding-top: $base-padding;
+}
+
+.landing-section,
+.landing-section_open {
+  @include respond-to(medium-up) {
+    @include span-columns(5.5);
+
+    border-bottom: none;
+  }
+
+  border-bottom: $thin-border-size solid $mid-gray;
+}
+
+.landing-section {
+  @include respond-to(medium-up) {
+    &:nth-child(even) {
+      @include shift(1);
+    }
+
+    &:nth-child(odd) {
+      @include omega();
+    }
+  }
+}
+
+.landing-section_open {
+  &.oil-gas {
+    background: url('../img/revenues/cards/oil-gas.png') bottom right no-repeat;
+  }
+
+  &.coal {
+    background: url('../img/revenues/cards/coal.png') bottom right no-repeat;
+  }
+
+  &.minerals {
+    background: url('../img/revenues/cards/minerals.png') bottom right no-repeat;
+  }
+
+  &.renewables {
+    background: url('../img/revenues/cards/renewables.png') bottom right no-repeat;
+  }
+
+  &.overview {
+    color: $mid-gray;
+
+    .landing-description {
+      @include span-columns(8);
+    }
+
+    .landing-icons {
+      @include span-columns(4);
+      @include font-size(3);
+      @include omega();
+
+      margin-bottom: $base-padding-lite;
+    }
+
+    .landing-steps {
+      @include span-columns(12);
+      font-size: 0.9em;
+
+      border: 1px solid $mid-gray;
+      padding: $base-padding-lite;
+      text-align: center;
+
+      icon {
+        color: $mid-gray;
+      }
+    }
+  }
+
+  &.oil-gas,
+  &.coal,
+  &.minerals,
+  &.renewables {
+    background-size: 100%;
+    height: 310px;
+
+    .landing-description {
+      @include span-columns(6);
+    }
+  }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 88 to appear before line 31. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets